src/index.html

```diff
@@ -516,6 +516,43 @@ <h1 class="clear">Map boundary change</h1>
       </div>
       <div id="main-map" class="map-area"></div>
     </div>
+    <div class="facility-info">
```

Comment:
can you switch those back. the rich data print view does not work anymore. might be related to your change here.